common/app/common/NewNavigation.scala

```diff
@@ -13,6 +13,7 @@ object NewNavigation {
 
   val PrimaryLinks = List(headlines, opinion, sport, culture, lifestyle)
   val topLevelSections = List(News, Opinion, Sport, Arts, Life)
+  val brandExtensions = List(jobs, dating, holidays, ukMasterClasses)
```

Comment:
Also, the campaign codes for this will change by edition so I don't think this is the right format for this afterall